fetool/pages/_app.js

```diff
@@ -1,7 +1,46 @@
-import '../styles/globals.css'
+import "../styles/globals.css";
+import { ToastContainer,toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { parseCookies  } from 'nookies'
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <>
+      <ToastContainer autoClose={1000}/>
```

Comment:
Isn't 1000 ms too less to show a toast message?

Reply:
Corrected to 3000ms